### PR TITLE
Review and change comments of the tuple match statement BBE

### LINKED ISSUE #13011: Proofread and correct the descriptions in tuple-match-statment example
$subject

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
This is to Review and change comments of the tuple match statement BBE.

Fixes #13011

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Required Balo version update
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: examples/tuple-match-statement/tuple_match_statement.bal [hunk 1]
```diff
@@ -28,28 +29,33 @@ public function main() {
...
-// will be checked for "isLike" relationship and will be matched at runtime.
+// This method uses tuple match patterns of different sizes. The `match` expression `a`
+// will be matched against the given pattern list at run time based on the 
+// "is-like" relationship between the expression and a pattern. 
+
+
 function basicMatch(any a) {
     match a {
-        // This pattern check is for a tuple type of three variables and types can be of any.
+        // This pattern check is for a tuple type of three variables of type 'any'.
```
Comment (pubudu91) [inline examples/tuple-match-statement/tuple_match_statement.bal]:
```suggestion
        // This pattern check is for a tuple type of three variables of type `any`.
```

File: examples/tuple-match-statement/tuple_match_statement.bal [hunk 2]
```diff
@@ -28,28 +29,33 @@ public function main() {
...
         var (s, i, b) => io:println("Matched with three vars : "
                                     + io:sprintf("%s", a));
-        // This pattern check is for a tuple type of two variables and types can be of any.
+        // This pattern check is for a tuple type of two variables of type 'any'.
```
Comment (pubudu91) [inline examples/tuple-match-statement/tuple_match_statement.bal]:
```suggestion
        // This pattern check is for a tuple type of two variables of type `any`.
```

File: examples/tuple-match-statement/tuple_match_statement.bal [hunk 3]
```diff
@@ -28,28 +29,33 @@ public function main() {
...
         var (s, i) => io:println("Matched with two vars : "
                                     + io:sprintf("%s", a));
-        // This pattern check is for a single variable which can be of type `any`. This has to be the last pattern.
+        // This pattern check is for a single variable of type 'any`. This has to be the last pattern.
```
Comment (pubudu91) [inline examples/tuple-match-statement/tuple_match_statement.bal]:
```suggestion
        // This pattern check is for a single variable of type `any`. This has to be the last pattern.
```

File: examples/tuple-match-statement/tuple_match_statement.bal (hunk 1 above)
Comment (praneesha) [reply examples/tuple-match-statement/tuple_match_statement.bal]: Fixed.

File: examples/tuple-match-statement/tuple_match_statement.bal (hunk 3 above)
Comment (praneesha) [reply examples/tuple-match-statement/tuple_match_statement.bal]: Fixed.

File: examples/tuple-match-statement/tuple_match_statement.bal (hunk 2 above)
Comment (praneesha) [reply examples/tuple-match-statement/tuple_match_statement.bal]: Fixed.

**Pull request merged**